Extensions/xkit_updates.js

```diff
@@ -292,7 +292,7 @@ XKit.extensions.xkit_updates = new Object({
 				try {
 					var mdata = JSON.parse(response.responseText);
 
-					if (mdata.malicious === true || mdata.malicious == "true") {
+					if (mdata.malicious || mdata.malicious == "true") {
```

Comment:
@ThePsionic this is not how javascript works. if(x) coerces the value to a boolean, and therefore if m.malicious was "false" it would coerce to boolean as true (because it is a non-empty string)